Misc/NEWS.d/next/Library/2019-10-24-08-10-30.bpo-38565.SWSUst.rst

```diff
@@ -0,0 +1 @@
+add new cache_parameters method for lru_cache
```

Comment:
``Add new cache_parameters() method for functools.lru_cache() to better support pickling.